File: Data Structures & Algorithms/two-integer-sum/submission-0.go
func twoSum(nums []int, target int) []int {
    size := len(nums)
    
    
    for i := 0; i < size; i++ {
        for j := i+1 ; j < size ; j++{
            sum := nums[i] + nums[j]
            if sum == target {
                return []int{i,j}
            }
        }
    }
    return nil
}

File: Data Structures & Algorithms/combination-target-sum/submission-0.py
class Solution:
    def combinationSum(self, nums: List[int], target: int) -> List[List[int]]:
        
        res = []

        def dfs(arr,count,numss):
            if count >= target:
                if count == target:
                    res.append(arr[:])
                return
            
            for i,num in enumerate(numss):
                arr.append(num)
                count = sum(arr)
                # print(arr,count)
                # print("_________________")
                dfs(arr,count,numss[i:])
                arr.pop()

        dfs([],0,nums)
        return res